inference-engine/tests/functional/plugin/shared/src/single_layer_tests/reduce_ops.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include <functional>
+#include <functional_test_utils/skip_tests_config.hpp>
+
+#include "ie_core.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+
+#include "single_layer_tests/reduce_ops.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string ReduceOpsLayerTest::getTestCaseName(testing::TestParamInfo<reduceMeanParams> obj) {
+    InferenceEngine::Precision netPrecision;
+    bool keepDims;
+    ngraph::helpers::ReductionType reductionType;
+    std::vector<size_t> inputShape;
+    std::vector<int> axes;
+    std::string targetDevice;
+    std::tie(axes, keepDims, reductionType, netPrecision, inputShape, targetDevice) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "axes=" << CommonTestUtils::vec2str(axes) << "_";
+    result << "type=" << reductionType << "_";
+    if (keepDims) result << "KeepDims_";
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice;
+    return result.str();
+}
+
+void ReduceOpsLayerTest::SetUp() {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
```

Comment:
Looks like extra line.

Reply:
Nope, this test fails on the setup stage since reduce operation can't be created with some specific parameters.